### PR TITLE
new: move api authorization policies to cid

### PR DESCRIPTION
**Pull request by t00f**
This PR is part of the first project to split the backend.

### CODE REVIEW

File: specs/apiauthorizationpolicy.spec [hunk 1]
```diff
@@ -27,26 +27,30 @@ model:
...
   - '@named'
   - '@hidden'
   - '@fallback'
   - '@schedulable'
   - '@timeable'
-
-# Indexes
-indexes:
-- - :no-inherit
+  - '@zoned'
```
Comment (primalmotion) [inline specs/apiauthorizationpolicy.spec:37]: I guess you basically miss all other indexes

**Review by primalmotion — DISMISSED** [review]

File: specs/apiauthorizationpolicy.spec [hunk 2]
```diff
@@ -27,26 +27,30 @@ model:
   - '@namespaced'
   - '@described'
   - '@disabled'
-  - '@identifiable-not-stored'
+  - '@identifiable-stored'
```
Comment (primalmotion) [inline specs/apiauthorizationpolicy.spec:30]: you should add propagated

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "6270d26d4874499a1ed31c49c235ed11bb49389d",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "0bbf055835fe0ffc8b5fc8f9c88496890d441b66",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "6270d26d4874499a1ed31c49c235ed11bb49389d",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "e062a775ff487a9b4e7a76c329daa45c7d33d81a",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "6270d26d4874499a1ed31c49c235ed11bb49389d",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "45c4773d3a8bfc7b8e56ed3c73cd4f13f4b0725e",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "6270d26d4874499a1ed31c49c235ed11bb49389d",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "e3b4af8e974bdf8e83498f57d7cf96e5581473ed",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "7617e7c5badf301dde0d65f7baa4e1f360e03787",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "6270d26d4874499a1ed31c49c235ed11bb49389d",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "8d76f5e3cb1193e85b3f0e2d56c333955ec9ee15",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "6270d26d4874499a1ed31c49c235ed11bb49389d",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "6270d26d4874499a1ed31c49c235ed11bb49389d",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "d5fd476d10d31a1204f20bebe64efdebdf25b4ef",
    "pipeline": "master"
  }
]
```

File: specs/apiauthorizationpolicy.spec [hunk 3]
```diff
@@ -27,26 +27,58 @@ model:
...
   - '@fallback'
+  - '@propagated'
   - '@schedulable'
   - '@timeable'
+  - '@zoned'
 
 # Indexes
 indexes:
-- - :no-inherit
+- - namespace
```
Comment (primalmotion) [inline specs/apiauthorizationpolicy.spec:42]: you don't have a type here.

File: specs/apiauthorizationpolicy.spec [hunk 4]
```diff
@@ -77,11 +111,21 @@ attributes:
     getter: true
     setter: true
 
+  - name: propagate
```
Comment (primalmotion) [inline specs/apiauthorizationpolicy.spec:114]: should be readonly to make it clear

**Review by primalmotion — CHANGES_REQUESTED** [review]

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "f1eebc081bb5414b5be98c8a7bc963b4172d89e2",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "ab435de6fa4bddf180f77fa188acc72c1007c567",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "f1eebc081bb5414b5be98c8a7bc963b4172d89e2",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "dc031bfdbf24c54a0a73c01f75d1b9531cb01b0e",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "move_apiauth",
    "component": "gaia",
    "pr-id": "654",
    "commit-sha": "30ed8d50ae1cfdb934dbbf8438fbb2a9cc96be68",
    "pipeline": "master"
  },
  {
    "project": "move_apiauth",
    "component": "backend",
    "pr-id": "1193",
    "commit-sha": "bc5f2c5a2a4d3ab4c8bc1f6c1081e8ceeadb2d4f",
    "pipeline": "master"
  }
]
```

**Pull request merged**